pootle/apps/pootle_data/directory_data.py

```diff
@@ -21,7 +21,7 @@ class DirectoryDataTool(RelatedStoresDataTool):
 
     @property
     def context_name(self):
-        return self.context.pootle_path
+        return self.context.pootle_path.strip("/").replace("/", ".")
```

Comment:
can you recall why it is necessary? 
`/af/prj0/dir.dir/` can't be distinct from `/af/prj/dir/dir/`